standards/SPIFFE_Federation.md

```diff
@@ -0,0 +1,316 @@
+5.2. [Endpoint Profiles](#52-endpoint-profiles)  
+5.2.1. [Web PKI (`https_web`)](#521-web-pki-https_web)  
+5.2.1.1. [Endpoint URL Requirements](#5211-endpoint-url-requirements)  
+5.2.1.2. [Endpoint Parameters](#5212-endpoint-parameters)  
+5.2.1.3. [Serving a Bundle Endpoint](#5213-serving-a-bundle-endpoint)  
+5.2.1.4. [Consuming a Bundle Endpoint](#5214-consuming-a-bundle-endpoint)  
+5.2.2. [SPIFFE Authentication (`https_spiffe`)](#522-spiffe-authentication-https_spiffe)  
+5.2.2.1. [Endpoint URL Requirements](#5221-endpoint-url-requirements)  
+5.2.2.2. [Endpoint Parameters](#5222-endpoint-parameters)  
+5.2.2.3. [Serving a Bundle Endpoint](#5223-serving-a-bundle-endpoint)  
+5.2.2.4. [Consuming a Bundle Endpoint](#5224-consuming-a-bundle-endpoint)  
+6\. [Relationship Lifecycle](#6-relationship-lifecycle)  
+6.1. [Establishing a Relationship](#61-establishing-a-relationship)  
+6.2. [Maintaining a Relationship](#62-maintaining-a-relationship)  
+6.3. [Terminating a Relationship](#63-terminating-a-relationship)  
+6.4. [Lifecycle Diagram](#64-lifecycle-diagram)  
+7\. [Security Considerations](#7-security-considerations)  
+7.1. [Distribution of Endpoint Parameters](#71-distribution-of-endpoint-parameters)  
+7.2. [Explicitly-defined Endpoint Parameters](#72-explicitly-defined-endpoint-parameters)  
+7.3. [Preserving the `<Trust Domain, Bundle>` Binding](#73-preserving-the-trust-domain-bundle-binding)  
+7.4. [Trustworthiness of the Bundle Endpoint Server](#74-trustworthiness-of-the-bundle-endpoint-server)  
+7.5. [Authenticity of the Bundle Endpoint](#75-authenticity-of-the-bundle-endpoint)  
+7.5.1. [Bundle Endpoint URL Redirection](#751-bundle-endpoint-url-redirection)  
+7.5.2. [Network Traffic Interception](#752-network-traffic-interception)  
+7.5.3. [Endpoint Parameters](#753-endpoint-parameters)  
+7.6. [Chaining Trust with `https_spiffe`](#76-chaining-trust-with-https_spiffe)  
+
+## 1. Background
+The SPIFFE specifications define the documents and interfaces necessary for establishing a platform-agnostic workload identity framework that is capable of bridging systems in different domains without the need for implementing identity translation or credential exchange logic. They define a “[trust domain][1]”, which serves as an identity namespace.
+
+SPIFFE is decentralized by nature. Each trust domain acts in its own capacity, under its own authority, and is administratively isolated from systems residing in other trust domains. Although trust domains delineate administrative and/or security domains, a core SPIFFE use case is enabling communication across these same boundaries where needed. Therefore, it is necessary to define a mechanism by which an entity may be introduced to a foreign trust domain, allowing it to authenticate identities issued by “other” SPIFFE authorities and allowing workloads in one trust domain to securely authenticate workloads in a foreign trust domain.
```

Comment:
Should it say "asserted" rather than "issued"? Since an authority may issue a credential that asserts an identity, but it doesn't really issue the identity itself.

Similarly in the paragraph below.

Apologies if this is standard terminology in the SPIFFE specs, I'm new to reading this. :)

Reply:
```suggestion
SPIFFE is decentralized by nature. Each trust domain acts in its own capacity, under its own authority, and is administratively isolated from systems residing in other trust domains. Although trust domains delineate administrative and/or security domains, a core SPIFFE use case is enabling communication across these same boundaries where needed. Therefore, it is necessary to define a mechanism by which an entity may be introduced to a foreign trust domain, allowing it to authenticate credentials issued by “other” SPIFFE authorities and allowing workloads in one trust domain to securely authenticate workloads in a foreign trust domain.
```